part-spec/common/componentID.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/componentID.json",
```

Comment:
I think we do want this to be edatasheets.github.io since that's where the release is? We should probably decide on this.